python/cuml/cuml/linear_model/logistic_regression.pyx

```diff
@@ -282,6 +241,45 @@ class LogisticRegression(UniversalBase,
         Fit the model with X and y.
 
         """
+        if self.penalty not in supported_penalties:
+            raise ValueError("`penalty` " + str(self.penalty) + " not supported.")
+
+        if self.solver not in supported_solvers:
+            raise ValueError("Only quasi-newton `qn` solver is "
+                             " supported, not %s" % self.solver)
+
+        if self.penalty == "none":
+            warnings.warn(
+                "The 'none' option was deprecated in version 24.06, and will "
+                "be removed in 25.08. Use None instead.",
+                FutureWarning
+            )
+            penalty = None
+        else:
+            penalty = self.penalty
+
+        if penalty == "elasticnet":
+            if self.l1_ratio is None:
+                raise ValueError(
+                    "l1_ratio has to be specified for" "loss='elasticnet'"
+                )
+            if self.l1_ratio < 0.0 or self.l1_ratio > 1.0:
+                msg = "l1_ratio value has to be between 0.0 and 1.0"
+                raise ValueError(msg.format(self.l1_ratio))
+
+        l1_strength, l2_strength = self._get_qn_params()
+        self.solver_model = QN(
+            loss="sigmoid",
+            fit_intercept=self.fit_intercept,
+            l1_strength=l1_strength,
+            l2_strength=l2_strength,
+            max_iter=self.max_iter,
+            linesearch_max_iter=self.linesearch_max_iter,
+            tol=self.tol,
+            verbose=self.verbose,
+            handle=self.handle,
```

Comment:
```suggestion
            handle=self.handle,
            output_type=self.output_type
```

If I'm not mistaken, that's what 99% of the pytest failures are complaining about